### PR TITLE
Travi-clang checks all files. Linux version error removed

### PR DESCRIPTION
**Pull request by CodeMaxx**
Addresses #902 #903 
@certik @isuruf

### CODE REVIEW

**Comment by CodeMaxx** [comment]
@certik Shouldn't we have a similar test like [this](https://github.com/symengine/symengine/blob/master/.travis.yml#L33) for OS X ?

File: symengine/polynomial.cpp [hunk 1]
```diff
@@ -27,8 +27,7 @@ UnivariateIntPolynomial::UnivariateIntPolynomial(
     SYMENGINE_ASSERT(is_canonical(degree_, dict_))
 }
 
-bool UnivariateIntPolynomial::is_canonical(const unsigned int &degree_,
-                                           const map_uint_mpz &dict) const
+bool UnivariateIntPolynomial::is_canonical(const unsigned int &degree_, const map_uint_mpz &dict) const
```
Comment (certik) [inline symengine/polynomial.cpp]: What caused this change?

File: bin/travis_clang.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ if [[ "${TRAVIS_OS_NAME}" == "linux" ]] && \
         exit 1
     fi
 
-    FILES=`git diff master --name-only | grep -E "\.(cpp|h)$"`
+    FILES=`grep -E "\.(cpp|h)$"`
```
Comment (abhinavagarwal07) [inline bin/travis_clang.sh]: This is better unchanged

**Comment by srajangarg** [comment]
@CodeMaxx I think this PR is not needed, the changes to clang-format have already been made.

**Comment by CodeMaxx** [comment]
@srajangarg Cool :+1:

**Pull request closed without merging**